### PR TITLE
fix: add default placeholder to time input

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/angular-core/styles/typography/_form.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@
   text-align: center;
 
   @include publicOnly() {
-    $timeInputWidth: 70;
+    $timeInputWidth: 84;
```
Comment (jeripeierSBB) [inline src/angular-core/styles/typography/_form.scss:42]: 84 as specified for public

File: src/angular-core/styles/typography/_form.scss [hunk 2]
```diff
@@ -57,7 +57,7 @@
   }
 
   @include businessOnly() {
-    width: 68px;
+    width: 70px;
```
Comment (jeripeierSBB) [inline src/angular-core/styles/typography/_form.scss:60]: should be 68px, but it's too narrow to display default placeholder (HH:MM)

File: src/angular-public/field/field.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-`<sbb-field>` works with `<input sbbInput>`, `<select sbbInput>`, `<sbb-select>` and `<input sbbInput sbbDateInput>`.
+`<sbb-field>` works with `<input sbbInput>`, `<textarea sbbInput>`, `<select sbbInput>`, `<sbb-select>`, `<input sbbInput sbbDateInput>`, `<input sbbInput sbbTimeInput>`.
```
Comment (jeripeierSBB) [inline src/angular-public/field/field.md:1]:
```suggestion
`<sbb-field>` works with `<input sbbInput>`, `<textarea sbbInput>`, `<select sbbInput>`, `<sbb-select>`, `<input sbbInput sbbDateInput>` and `<input sbbInput sbbTimeInput>`.
```

File: src/angular-public/time-input/time-input/time-input.directive.ts [hunk 4]
```diff
@@ -5,30 +5,54 @@ import {
...
+   * placeholder (defaults to HH:MM)
+   */
+  @Input()
+  @HostBinding('attr.placeholder')
+  get placeholder(): string {
+    return this._placeholder;
+  }
+  set placeholder(value: string) {
+    this._placeholder =
+      typeof value === 'undefined' || value === null ? PLACEHOLDER_DEFAULT : value;
```
Comment (kyubisation) [inline src/angular-public/time-input/time-input/time-input.directive.ts:35]: You could use the ?? operator here, which is equivalent to the undefined/null check.
```suggestion
    this._placeholder = value ?? PLACEHOLDER_DEFAULT;
```

**Pull request merged**